### ISSUE TITLE
Support refresh tokens

### ISSUE THREAD

**Issue by andymckay-limited-access** [mozilla/payments#97]
As per emails to dev-fxa, refresh tokens are now a thing. This issue is slightly exploratory. Do we need to do anything with it? Should we expect to receive a refresh token or an access token?

If we need to do anything to support refresh tokens, let's hang them off this bug.

**Comment by kumar303** [comment]
I don't think we need to do anything but I filed this a while back to at least show how to use them in the example app: https://github.com/mozilla/payments-example/issues/5

**Comment by kumar303** [comment]
Unassigning myself for now since my queue is pretty big. I'll grab it later.
